extension/json/src/json_scan.cpp

```diff
@@ -8,25 +12,62 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace json_extension {
 
-struct JsonBindData : public TableFuncBindData {
+struct JsonScanConfig {
+    std::string format;
```

Comment:
Better to use enum instead of std::string, what are the possible options for enum?

Reply:
That's true. I'll change it to enum. The options are 'unstructed' and 'array' right now.